hub/scripts.js

```diff
@@ -187,64 +187,6 @@
     return config;
   }
 
-  /**
```

Comment:
All of this shouldn't be used anymore. 
Should be solved in the [helix-pipeline-service](https://github.com/adobe/helix-pipeline-service/blob/main/src/steps/render.js) @tripodsan or @kptdobe could you confirm that's true?

Reply:
Seems to work, but Narcis already removed the same thing in a branch that will land before the helix 3 migration in the `main` branch https://github.com/narcis-radu/fedpub/blob/blocks/hub/scripts.js

We'll ship removing the outdated code a bit later to make it easier for our QA to test the changes due to limited bandwidth before max